ray-operator/controllers/ray/common/service.go

```diff
@@ -131,11 +167,11 @@ func getPortsFromCluster(cluster rayiov1alpha1.RayCluster) (map[string]int32, er
 
 func getDefaultPorts() map[string]int32 {
 	return map[string]int32{
-		DefaultClientPortName:               DefaultClientPort,
-		DefaultRedisPortName:                DefaultRedisPort,
-		DefaultDashboardName:                DefaultDashboardPort,
-		DefaultMetricsName:                  DefaultMetricsPort,
-		DefaultDashboardAgentListenPortName: DefaultDashboardAgentListenPort,
```

Comment:
remove DefaultDashboardAgentListenPortName since it is controlled by annotation.